### DISCUSSION TITLE
Adding a cooldown / delay to a material / ItemStack similar to the shield item

### DISCUSSION THREAD

**Discussion by KianBrose** [Minestom/Minestom#1789] [Help] [answered]
As the title says, how can I add the effect that shields have in vanilla minecraft where it's on cooldown for a few seconds?

**Comment by GoldenCookieGIT** [comment, marked as answer]
You can send the player a cooldown packet. To create one of those you can use [SetCooldownPacket(materialId, ticks)](https://javadoc.minestom.net/net/minestom/server/network/packet/server/play/SetCooldownPacket.html#%3Cinit%3E(int,int))
